File: LLM/tests/fixtures/sample.go
package sample

import (
	"fmt"
	"strings"
)

// Greet returns a personalised greeting.
func Greet(name string) string {
	return fmt.Sprintf("Hello, %s!", name)
}

// Process processes a slice of strings and returns a result map.
func Process(data []string, verbose bool) map[string]interface{} {
	result := Greet("world")
	_ = strings.Join(data, ",")
	return map[string]interface{}{"processed": result, "count": len(data)}
}

// Add returns the sum of two integers.
func Add(a, b int) int {
	return a + b
}

// Compute delegates to Add.
func Compute(x int) int {
	return Add(x, 1)
}

File: LLM/tests/fixtures/Sample.java
import java.util.List;
import java.util.Map;
import java.util.HashMap;

/**
 * Sample Java class for AST parser tests.
 */
public class Sample {

    /**
     * Return a personalised greeting.
     */
    public static String greet(String name) {
        return "Hello, " + name + "!";
    }

    public static Map<String, Object> process(List<String> data, boolean verbose) {
        String result = greet("world");
        Map<String, Object> out = new HashMap<>();
        out.put("processed", result);
        out.put("count", data.size());
        return out;
    }

    /**
     * Simple calculator with basic arithmetic.
     */
    static class Calculator {

        /**
         * Return the sum of a and b.
         */
        public int add(int a, int b) {
            return a + b;
        }

        public int subtract(int a, int b) {
            return a - b;
        }
    }
}
